content/2_Bronze/Intro_Sets.mdx

```diff
@@ -1226,4 +1226,222 @@ make their implementation easier.
 
 </Quiz>
 </JavaSection>
+
+
+<PySection>
+
+<Quiz>
+	<Quiz.Question>
+		What is the time complexity of insertions, deletions, and searches in a sorted set of size $N$?
+		<Quiz.Answer>
+			$\mathcal{O}(1)$
+		</Quiz.Answer>
+		<Quiz.Answer>
+			$\mathcal{O}(N \log N)$
+		</Quiz.Answer>
+		<Quiz.Answer>
+			$\mathcal{O}(N)$
+		</Quiz.Answer>
+		<Quiz.Answer correct>
+			$\mathcal{O}(\log N)$
+			<Quiz.Explanation>
+				Correct.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+	</Quiz.Question>
+	<Quiz.Question>
+		Which of the following is the correct output of this code?
+		```py
```

Comment:
are languagesections not supported in quiz questions?

Reply:
<img width="41" alt="image" src="https://github.com/cpinitiative/usaco-guide/assets/81596991/0cfc4eb1-2779-4166-8141-ecdc6160205d">

nope, messes up answer selection cuz they all have diff questions and they must be directly under the question comp

+ it's prolly easier to edit if the quizzes are separated by language 

—--

you also can't have language sections underneath a <quiz> tag

Reply:
fff alright then